### PR TITLE
[Keyboard] Add yandrstudio/nz67v2 keyboard.

### PR DESCRIPTION
**Pull request by jiaxin96**
## Description
add nz67v2 keyboard.
`RESET` => `QK_BOOT` in readme of nz64 and zhou65.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Review by keyboard-magpie — CHANGES_REQUESTED** [review]
Is there any reason you have not included an info.json?

**Comment by jiaxin96** [comment]
> 1

The info.json file has been supplemented according to the qmk format requirements. Please review it again.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/yandrstudio/nz67v2/info.json [hunk 1]
```diff
@@ -0,0 +1,381 @@
+{
+    "keyboard_name": "nz67v2",
+    "processor": "STM32F103",
+    "usb": {
+        "pid": "0xAA83",
+        "device_version": "1.0.0"
+    },
```
[the remark this answers is not in the document]
Comment (jiaxin96) [reply keyboards/yandrstudio/nz67v2/info.json:7]: I wrote these common configurations in the root directory of thess keyboards (). Do I need to repeat them again in a specific keyboard?

File: keyboards/yandrstudio/nz67v2/info.json [hunk 2]
```diff
@@ -0,0 +1,384 @@
+{
+    "keyboard_name": "nz67v2",
+    "manufacturer": "Y&R",
+    "processor": "STM32F103",
+    "bootloader": "stm32duino",
+    "usb": {
+        "vid": "0xAA96",
```
Comment (fauxpark) [inline keyboards/yandrstudio/nz67v2/info.json:7]:
```suggestion
    "processor": "STM32F103",
    "bootloader": "stm32duino",
    "usb": {
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by keyboard-magpie — APPROVED** [review]

**Pull request merged**